### DISCUSSION TITLE
Error while running npm install in server directory

### DISCUSSION THREAD

**Discussion by Akash1965** [hereisSwapnil/ExamTime#214] [Q&A]
On running `npm install` inside the server directory as mentioned in the README.md, I am experiencing the following error:-

<img width="501" alt="Screenshot 2024-05-20 at 2 42 43 AM" src="https://github.com/hereisSwapnil/ExamTime/assets/115504480/7406f16d-4664-4a8c-9582-6fdc4552ccf4">

Can anyone help me with this issue
@hereisSwapnil

**Comment by DonaldReddy** [comment]
Hey @Akash1965 I too faced the same issue, what I did was downgrade the node to [v18.20.2](https://nodejs.org/en/download/prebuilt-installer) and npm to 10.8.0 by using this command `npm i -g npm@10.8.0`.

I hope this works for you 🤞.

**Reply by Akash1965** [reply]
I downgrade my node to v18.20.2 but when I am running `npm i -g npm@10.8.0` it's giving me the following error
<img width="503" alt="Screenshot 2024-05-20 at 12 21 04 PM" src="https://github.com/hereisSwapnil/ExamTime/assets/115504480/ddbb00c6-5ba3-41d2-bed7-ab92e90a21aa">
<img width="479" alt="Screenshot 2024-05-20 at 12 21 13 PM" src="https://github.com/hereisSwapnil/ExamTime/assets/115504480/ed17513d-6df5-4908-a461-b94a015e9fda">
<img width="485" alt="Screenshot 2024-05-20 at 12 21 24 PM" src="https://github.com/hereisSwapnil/ExamTime/assets/115504480/70c7d1b9-2a4e-4a71-97a4-17c037accd13">

**Comment by DonaldReddy** [comment]
I tried my best to solve the issue, but I was not able to find any solution. [maybe this will solve it](https://chatgpt.com/share/036ec8d5-de8e-40a1-9a64-6207e57fc893)

**Reply by akshita119** [reply]
how did u downgrade node

**Reply by DonaldReddy** [reply]
delete the current version and download the required version from the node.js official website
